src/testUtils.ts

```diff
@@ -195,22 +213,46 @@ export function goTest(testconfig: TestConfig): Thenable<boolean> {
 			errBuf.onLine(line => outputChannel.appendLine(expandFilePathInOutput(line, testconfig.dir)));
 			errBuf.onDone(last => last && outputChannel.appendLine(expandFilePathInOutput(last, testconfig.dir)));
 
-			proc.stdout.on('data', chunk => outBuf.append(chunk.toString()));
-			proc.stderr.on('data', chunk => errBuf.append(chunk.toString()));
+			tp.stdout.on('data', chunk => outBuf.append(chunk.toString()));
+			tp.stderr.on('data', chunk => errBuf.append(chunk.toString()));
 
-			proc.on('close', code => {
+			statusBarItem.show();
+
+			tp.on('close', (code, signal) => {
 				outBuf.done();
 				errBuf.done();
 
 				if (code) {
-					outputChannel.appendLine(`Error: ${testType} failed.`);
+					outputChannel.appendLine('Error: Tests failed.');
+				} else if (signal === sendSignal) {
```

Comment:
Any reason for removing the use of the `testType` variable?

Reply:
No reason. Not sure why this got removed.